examples/cookie-game/src/state_types.ts

```diff
@@ -0,0 +1,132 @@
+import { actions } from "./actions"
+
+/*
+   Understanding rule of game:
+   https://cookieclicker.fandom.com/wiki/Building
+*/
+
+export const init_cursorCps: number = 0.1;
+export const init_grandmaCps: number = 1;
+export const init_farmCps: number = 8;
+export const init_mineCps: number = 47;
+export const init_factoryCps: number = 260;
+export const init_bankCps: number = 1400;
+export const init_templeCps: number = 7800;
+export const init_wizardCps: number = 44_000;
+export const init_shipmentCps: number = 260_000;
+export const init_alchemyCps: number = 1.6e5;
+export const init_portalCps: number = 1e7;
+export const init_timeMachineCps: number = 65e6;
+export const init_antimatterCps: number = 43e7;
+export const init_prismCps: number = 2.9e8;
+export const init_chanceMakerCps: number = 22e9;
+export const init_fractalCps: number = 15e10;
+export const init_javascriptCps: number = 1.1e11;
+export const init_idleverseCps: number = 8.3e11;
+export const init_cordexCps: number = 64e12;
+
+export const init_cursorCost: number = 15;
+export const init_grandmaCost: number = 100;
+export const init_farmCost: number = 1100;
+export const init_mineCost: number = 12_000;
+export const init_factoryCost: number = 130_000;
+export const init_bankCost: number = 1_400_000;
+export const init_templeCost: number = 2e7;
+export const init_wizardCost: number = 33e7;
+export const init_shipmentCost: number = 5.1e9;
+export const init_alchemyCost: number = 75e9;
+export const init_portalCost: number = 1e12;
+export const init_timeMachineCost: number = 14e12;
+export const init_antimatterCost: number = 17e13;
+export const init_prismCost: number = 2.1e14;
+export const init_chanceMakerCost: number = 26e15;
+export const init_fractalCost: number = 31e16;
+export const init_javascriptCost: number = 71e18;
+export const init_idleverseCost: number = 12e21;
+export const init_cordexCost: number = 1.9e23;
+
+export type cookieBakerType = {
+    numberOfCookie: number;
```

Comment:
I think we should use shorter names, like `cookies`, `cursors`, etc.
Writing `numberOfCookies` is kinda really long for nothing interesting...